### PR TITLE
how to become a maintainer doc

### PR DESCRIPTION
**Pull request by amarie401**
This PR introduces a update to `contributing.md` with documentation on how to become a maintainer with slight modification for Patternfly NG. This is related to this [PR](https://github.com/patternfly/patternfly-react/pull/302) from @LHinson 

@dgutride @dlabrecq

### CODE REVIEW

**Review by LHinson — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -145,6 +145,57 @@ feat(pencil): add 'graphiteWidth' option
 perf(pencil): remove graphiteWidth option
 ```
 
+## Becoming a Maintainer
```
Comment (dlabrecq) [inline CONTRIBUTING.md]: There is a table of contents at the top of the document. Please add a link to this section.

See the commits link for reference.

`## <a name="commits"></a> Git Commit Guidelines`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -145,6 +145,57 @@ feat(pencil): add 'graphiteWidth' option
...
+- Make well formed pull requests.
+- Have a sense of duty about the project.
+- Play well with others, are respectful, show gratitude.
+
+If you want to become a maintainer, we expect you to:
+
+- Review and test pull requests submitted by others.
+- Encourage and ensure design remains an integral part of the review process and pull in designers for review as needed (you can leverage @patternfly/patternfly-ng-ux if there is no known associated designer).
+- Maintain sustained activity versus sporadic.
+- Support users and other developers on [PatternFly Slack](https://patternfly.slack.com/) (there is a channel dedicated to Patternfly NG) and the [mailing list](mailto:patternfly@redhat.com).
```
Comment (dlabrecq) [inline CONTRIBUTING.md]: We already provide Slack information at the top of this doc.

**Review by dlabrecq — CHANGES_REQUESTED** [review]
Can we link to this document somewhere on Patternfly? Perhaps even http://www.patternfly.org/community? For example, this page already provides links our "Code of Conduct", "Angular style guide", etc. Seems a lot of this is boiler plate and will be the same for all. I'm thinking this may be better maintained in one place Vs across multiple repos?

**Comment by LHinson** [comment]
@dlabrecq there is some changes happening to our site and some information will be moved around. Let's revisit updating the site with these links once those changes have been implemented.

File: CONTRIBUTING.md (hunk 2 above)
Comment (amarie401) [reply CONTRIBUTING.md]: Should I just remove line 167 since slack and mail information is already provided or just reword? @LHinson

Comment (LHinson) [reply CONTRIBUTING.md]: The line is about the responsibility to support these channels as a maintainer vs just providing the the links for those things. 

You can just remove the links so it reads
>Support users and other developers on the PatternFly slack channels and mailing list.

**Comment by amarie401** [comment]
changes have been updated @LHinson @dlabrecq

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -145,6 +146,57 @@ feat(pencil): add 'graphiteWidth' option
...
+- Review and merge code and documentation.
+- Help triaging bugs and testing pull requests.
+- Make well formed pull requests.
+- Have a sense of duty about the project.
+- Play well with others, are respectful, show gratitude.
+
+If you want to become a maintainer, we expect you to:
+
+- Review and test pull requests submitted by others.
+- Encourage and ensure design remains an integral part of the review process and pull in designers for review as needed (you can leverage @patternfly/patternfly-ng-ux if there is no known associated designer).
```
Comment (dlabrecq) [inline CONTRIBUTING.md]: What is @patternfly/patternfly-ng-ux? Is this a Slack channel, repo?

**Review by dlabrecq — CHANGES_REQUESTED** [review]
Just one more question...

File: CONTRIBUTING.md (hunk 3 above)
Comment (amarie401) [reply CONTRIBUTING.md]: I think this is the group of designers designated for the patternfly-ng repo for reviews if I'm not mistaken @LHinson 

https://github.com/orgs/patternfly/teams/patternfly-ng-ux

Comment (LHinson) [reply CONTRIBUTING.md]: you're correct @amarie401 :)

Comment (dlabrecq) [reply CONTRIBUTING.md]: Ok, can we add that URL here?

Comment (amarie401) [reply CONTRIBUTING.md]: @dlabrecq I have added the URL for the pf-ng design team 👍

**Review by dlabrecq — APPROVED** [review]
Thank you for the changes!

**Comment by amarie401** [comment]
Thanks @dlabrecq :)

**Pull request merged**